concrete/src/Captcha/SecurimageController.php

```diff
@@ -19,61 +48,99 @@ public function __construct()
         $this->securimage->num_lines = 5;
 
         $this->securimage->use_multi_text = true;
-        $this->securimage->multi_text_color = array(
+        $this->securimage->multi_text_color = [
             new Securimage_Color(184, 4, 50),
             new Securimage_Color(12, 67, 157),
             new Securimage_Color(244, 49, 11),
-        );
+        ];
         $this->securimage->text_color = new Securimage_Color(184, 4, 50);
     }
 
     /**
-     * Display the captcha.
+     * {@inheritdoc}
      */
-    public function display()
-    {
-        $ci = Loader::helper('concrete/urls');
-        echo '<div><img src="' . $ci->getToolsURL('captcha') . '?nocache=' .time(). '" alt="' .t('Captcha Code'). '" onclick="this.src = \'' . $ci->getToolsURL('captcha') . '?nocache=\'+(new Date().getTime())" class="ccm-captcha-image" /></div>';
-    }
-
-    public function label()
+    public function display(array $customImageAttributes = [])
```

Comment:
This is a backwards incompatible change.

Reply:
We can add another method. Any suggestion for its name?

Reply:
not sure on naming, but we need to remove the signature changes from each of these methods.